native/cocos/physics/physx/PhysXWorld.cpp

```diff
@@ -315,5 +329,46 @@ RaycastResult &PhysXWorld::raycastClosestResult() {
     return hit;
 }
 
+uint32_t PhysXWorld::addPXObject(uintptr_t PXObjectPtr) {
+    static uint32_t sPXObjectID = 0;
+    uint32_t PXobjectID = sPXObjectID;
+    sPXObjectID++;
+    assert(sPXObjectID < 0xffffffff);
+
+    _mPXObjects[PXobjectID] = PXObjectPtr;
+    return PXobjectID;
+};
+
+void PhysXWorld::removePXObject(uint32_t PXobjectID) {
+    _mPXObjects.erase(PXobjectID);
+}
+
+uintptr_t PhysXWorld::getPXPtrWithPXObjectID(uint32_t PXObjectID) {
+    if (_mPXObjects.find(PXObjectID) == _mPXObjects.end())
+        return 0;
+    return _mPXObjects[PXObjectID];
```

Comment:
I think can just cache the iterator for return value like this
```c++
const iter = _mPXObjects.find(PXObjectID);
if (iter == _mPXObjects.end()) {
    return;
}
return iter.second;
```

Reply:
Seems good